tests/snapisol.test/t6_01.req

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE newsqlresetbug (i INTEGER)$$
+SET TRANSACTION SNAPSHOT ISOLATION
+BEGIN
+UPDATE newsqlresetbug SET i = i WHERE 1=1
+COMMIT
+BEGIN
+SELECT * FROM newsqlresetbug
```

Comment:
Just curious, shouldn't these test req files be kept logically independent? So that they create objects and tear them down at the end.